entrypoint.sh

```diff
@@ -1313,6 +1313,17 @@ function print_loaded_drv_ver_str() {
     fi
 }
 
+function update_ca_certificates() {
+    timestamp_print "Updating system CA certificates (Ubuntu)..."
```

Comment:
Please move the OS check here, so that all platforms will eventually be in the same place